base_layer/core/src/consensus/emission.rs

```diff
@@ -92,6 +92,12 @@ impl EmissionSchedule {
     }
 }
 
+impl Default for EmissionSchedule {
+    fn default() -> Self {
+        Self::new(10_000_000.into(), 0.999, 100.into())
```

Comment:
Do we want these inside of constants? 
I am of two minds here, in one side this is definitely constants that define the network consensus rules, but in the other side it might clutter the constants struct with data that does not completely belong as they do not directly influence the network constants but rather indirectly by changing the emission curve.

Reply:
Not sure if this is what @SWvheerden was saying, but I'm not sure we want there to be a Default impl of EmissionSchedule. Because the question: "What is the default ES?" doesn't have a single answer.

Reply:
The default was removed and the configuration of the EmissionSchedule was moved to the ConsensusConstants.